app/controllers/catalog_controller.rb

```diff
@@ -4,9 +4,15 @@
 class CatalogController < ApplicationController
   include Blacklight::Catalog
 
-  # This overrides default behavior from the Blacklight::Catalog module in the
-  # blacklight gem. Method defined below.
-  rescue_from Blacklight::Exceptions::RecordNotFound, with: :record_not_found
+  # This shouldn't be necessary, since it's also specified in ApplicationController...
+  # but I must be missing something.
```

Comment:
@mccalluc there is a [`rescue_from` statement coming from Blacklight::Catalog](https://github.com/projectblacklight/blacklight/blob/master/app/controllers/concerns/blacklight/catalog.rb#L18) that will override whatever you specify in `ApplicationController`, so you need to re-override it here. Not ideal, but meh.